apps/judicial-system/web/src/routes/CourtOfAppeal/components/CaseNumbers/CaseNumbers.tsx

```diff
@@ -10,15 +10,23 @@ const CaseNumbers: FC = () => {
   const { workingCase } = useContext(FormContext)
   const { formatMessage } = useIntl()
 
+  if (!workingCase.appealCaseNumber) {
+    return (
+      <Text as="h3" variant="default" fontWeight="semiBold">
+        {formatMessage(strings.courtOfAppealCaseNumber, {
+          caseNumber: workingCase.courtCaseNumber,
+        })}
+      </Text>
+    )
+  }
+
   return (
     <Box marginBottom={7}>
-      {workingCase.appealCaseNumber && (
-        <Text as="h2" variant="h2">
-          {formatMessage(strings.caseNumber, {
-            caseNumber: `${workingCase.appealCaseNumber}`,
-          })}
-        </Text>
-      )}
+      <Text as="h2" variant="h2">
+        {formatMessage(strings.caseNumber, {
+          caseNumber: `${workingCase.appealCaseNumber}`,
+        })}
+      </Text>
```

Comment:
<img width="812" alt="Screenshot 2025-05-08 at 13 27 19" src="https://github.com/user-attachments/assets/fdfa0e0f-5274-46c7-b251-f6a41c66aa11" />
Felt this was a bit awkward so i added when both numbers it looks like this 
<img width="800" alt="Screenshot 2025-05-08 at 13 27 42" src="https://github.com/user-attachments/assets/604ab950-1dbf-44c3-adec-ec72dd64b545" />
<img width="811" alt="Screenshot 2025-05-08 at 14 14 55" src="https://github.com/user-attachments/assets/1caa4a96-be0a-498e-9a57-60c1ce35b51a" />